### PR TITLE
docs: update the release schedule with v6 info

### LINKED ISSUE #20649: Release Schedule needs an update
## I'm submitting a...

<pre><code>
[ ] Regression (a behavior that used to work and stopped working in a new release)
[ ] Bug report  
[ ] Feature request
[x] Documentation issue or request
[ ] Support request => Please do not submit support request here, instead see https://github.com/angular/angular/blob/master/CONTRIBUTING.md#question
</code></pre>

## Current behavior

[Release Schedule](https://github.com/angular/angular/blob/master/docs/RELEASE_SCHEDULE.md) documentation is old and outdated with the v5 release now done.

## Expected behavior

It should be updated with the 5.1+ release schedule.

## Minimal reproduction of the problem with instructions

[Click here](https://github.com/angular/angular/blob/master/docs/RELEASE_SCHEDULE.md)

## What is the motivation / use case for changing the behavior?

So people know when to expect 5.1+.

## Environment

<pre><code>
Angular version: X.Y.Z

Browser:
- [ ] Chrome (desktop) version XX
- [ ] Chrome (Android) version XX
- [ ] Chrome (iOS) version XX
- [ ] Firefox version XX
- [ ] Safari (desktop) version XX
- [ ] Safari (iOS) version XX
- [ ] IE version XX
- [ ] Edge version XX
 
For Tooling issues:
- Node version: XX  
- Platform:  

Others:

</code></pre>

### PR DESCRIPTION
**Pull request by IgorMinar**
Fixes #20649

### CODE REVIEW

File: docs/RELEASE_SCHEDULE.md [hunk 1]
```diff
@@ -32,41 +32,28 @@ If you see undesirable wrapping issues in the rendered form, please copy&paste t
...
+2018-04-11	  | 6.0.2                               | -                                  |
+2018-04-18	  | -                                   | -                                  | [ng-conf](https://www.ng-conf.org/)
 
 ## Tentative Schedule After September 2017
 
  Date                   | Stable Release | Compatibility`*`
  ---------------------- | -------------- | ----------------
- March/April 2018       | 6.0.0          | ^5.0.0
  September/October 2018 | 7.0.0          | ^6.0.0
+ March/April 2018       | 8.0.0          | ^7.0.0
```
Comment (matpag) [inline docs/RELEASE_SCHEDULE.md]: Wasn't this line already fine? March/April 2018 is for the 6.0.0 release

Comment (skreborn) [reply docs/RELEASE_SCHEDULE.md]: I suppose that was meant to read _"March/April 2019"_.

Comment (matpag) [reply docs/RELEASE_SCHEDULE.md]: Oh ok 😉

Comment (IgorMinar) [reply docs/RELEASE_SCHEDULE.md]: aha! good catch. I'll fix. thanks

Comment (IgorMinar) [reply docs/RELEASE_SCHEDULE.md]: fixed

**Review by mhevery — APPROVED** [review]

**Pull request closed without merging**